app/controllers/casa_cases_controller.rb

```diff
@@ -190,11 +190,11 @@ def changed_attributes_messages(original, changed)
 
   def change_message_text(attribute, original_attribute, updated_attribute)
     if attribute == :contact_types
-      new_contact_type_ids = updated_attribute.map { |contact| contact["contact_type_id"] }
-      previous_contact_type_ids = original_attribute.map { |contact| contact["contact_type_id"] }
-      changed_count = new_contact_type_ids - previous_contact_type_ids
-      return if changed_count == 0
-      "#{changed_count} #{attribute.to_s.humanize.singularize.pluralize(changed_count)} added"
+      new_contact_types = updated_attribute.map { |contact| contact["name"] }
+      previous_contact_types = original_attribute.map { |contact| contact["name"] }
+      changed_contact_types = new_contact_types - previous_contact_types
+      return if changed_contact_types.empty?
+      "#{changed_contact_types} #{attribute.to_s.humanize.singularize.pluralize(changed_contact_types)} added"
```

Comment:
- changed variables names as I replaced IDs with names.
- I was not able to figure out how to test this. Any particular examples on how to write a unit test on this private method?